trivy-task/trivyV2/task.json

```diff
@@ -0,0 +1,213 @@
+  "preview": true,
+  "description": "Trivy is the world's most popular open source vulnerability and misconfiguration scanner. It is reliable, fast, extremely easy to use, and it works wherever you need it.",
+  "helpMarkDown": "[Learn more about this task](https://github.com/aquasecurity/trivy-azure-pipelines-task)",
+  "helpUrl": "https://github.com/aquasecurity/trivy-azure-pipelines-task",
+  "category": "Test",
+  "author": "Aqua Security",
+  "version": {
+    "Major": 2,
+    "Minor": 0,
+    "Patch": 0
+  },
+  "instanceNameFormat": "Echo trivy $(version)",
+  "groups": [
+    {
+      "name": "runnerInput",
+      "displayName": "Trivy Runner",
+      "isExpanded": true
+    },
+    {
+      "name": "scanInput",
+      "displayName": "Scan Options",
+      "isExpanded": true
+    },
+    {
+      "name": "reportsInput",
+      "displayName": "Reports",
+      "isExpanded": true
+    }
+  ],
+  "inputs": [
+    {
+      "name": "aquaPlatform",
+      "type": "connectedService:AquaPlatform",
+      "label": "Aqua Security Platform",
+      "defaultValue": "",
+      "required": false,
+      "helpMarkDown": "Select the Aqua Security Platform service connection."
+    },
+    {
+      "groupName": "runnerInput",
+      "name": "method",
+      "type": "pickList",
+      "label": "Trivy Run Method",
+      "defaultValue": "install",
+      "required": false,
+      "helpMarkDown": "Specify how Trivy should be executed:</br>`install` to download Trivy from GitHub releases</br>`docker` to run Trivy as Docker container</br>`system` to use pre-installed Trivy executable.",
+      "options": {
+        "install": "install",
+        "docker": "docker",
+        "system": "system"
+      }
+    },
+    {
+      "groupName": "runnerInput",
+      "name": "image",
+      "type": "string",
+      "label": "Trivy Docker Image",
+      "defaultValue": "",
+      "required": false,
+      "helpMarkDown": "Specify a custom Trivy Docker image to use. If set, the `version` option is ignored.",
+      "visibleRule": "method = docker"
+    },
+    {
+      "groupName": "runnerInput",
+      "name": "version",
+      "type": "string",
+      "label": "Trivy Version",
+      "defaultValue": "latest",
+      "required": false,
+      "helpMarkDown": "Specify the version of Trivy to use. Ignored if a custom Trivy Docker image is specified.",
+      "visibleRule": "method != system || image != \"\""
+    },
+    {
+      "groupName": "runnerInput",
+      "name": "option",
+      "type": "string",
+      "label": "Trivy Options",
+      "defaultValue": "",
+      "required": false,
+      "helpMarkDown": "Provide additional command-line options to pass to the Trivy executable."
+    },
+    {
+      "groupName": "scanInput",
+      "name": "type",
+      "type": "pickList",
+      "label": "Scan Type",
+      "required": true,
+      "helpMarkDown": "The type of scan to perform.",
+      "options": {
+        "filesystem": "filesystem",
```

Comment:
We are focusing on `filesystem` and `image` for now. k8s and vm aren't something we want to support in the pipeline task. 

SBOM seems like it could potentially be useful but we need to ensure that the UI doesn't need changes.

For now, please reduce the list to `filesystem` and `image`